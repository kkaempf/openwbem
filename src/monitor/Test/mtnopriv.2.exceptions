InsufficientPrivilegesException  CheckException: rename: insufficient privileges
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
