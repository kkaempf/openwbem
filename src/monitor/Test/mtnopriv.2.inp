rename /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/rename1 /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/rename-to
rename /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/rename1 /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/rename-to
