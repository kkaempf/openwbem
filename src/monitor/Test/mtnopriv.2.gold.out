User: owprovdr

rename
Exception:
  type: PrivilegeManagerException
  msg:  CheckException: rename: insufficient privileges

rename
Exception:
  type: IPCIOException
